.gitignore

```diff
@@ -25,8 +25,8 @@
 *.rbc
 *.seed
 
-**/__snapshots__/diff
-**/__snapshots__/latest
+**/__snapshots__/*
+!**/__snapshots__/reference/
```

Comment:
May I know why are you ignoring reference images?

Reply:
The ! means to not ignore. So what this change does is ignore all folders within snapshots except for reference.